components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/api/client/GradleEnterpriseApiClient.java

```diff
@@ -132,7 +133,8 @@ public BuildValidationData fetchBuildValidationData(String buildScanId) {
                     attributes.getRequestedGoals(),
                     buildOutcomeFrom(attributes),
                     remoteBuildCacheUrlFrom(buildCachePerformance),
-                    summarizeTaskExecutions(buildCachePerformance)
+                    summarizeTaskExecutions(buildCachePerformance),
+                    toDuration(buildCachePerformance.getEffectiveProjectExecutionTime())
```

Comment:
Same comment as above.

Reply:
I have done that here on a new branch: https://github.com/gradle/gradle-enterprise-build-validation-scripts/commit/fba624e56e205bd8e1a36ae0960be7218a1c7e46